docs-src/modules/getting-started/pages/hello-world-cli.adoc

```diff
@@ -309,6 +296,16 @@ When you configure `--preview-rows` Kaskada stops processing at the end of a bat
 ⭢  Read more about configuring queries in xref:developing:queries.adoc#configuring-how-queries-are-computed[Configuring Queries]
 ****
 
+== Cleaning Up
+
+When you're done with this tutorial, you can delete the table you created in order to free up resources.  Note that this also deletes all of the data loaded into the table.
+
+[source,bash]
+----
+# Delete the Purchase table
+kaskada-cli table delete --table Purchase
+----
+
```

Comment:
cleanup was dropped in previous PR, but I think it should be included.  Most getting-started docs I've seen include help to reset your state to a "new" system.